src/main/java/org/walkmod/javalang/compiler/types/TypesLoaderVisitor.java

```diff
@@ -159,7 +164,7 @@ private String getSymbolName(String name, boolean imported) {
                simpleName = aux;
             }
          }
-         simpleName = simpleName.replaceAll("\\$", ".");
+         simpleName = simpleName.replace("$", ".");
```

Comment:
I just replaced the unneeded regexp replace with simple replace. The \\ makes $ a normal character. No functional change.